README.md

```diff
@@ -79,7 +79,7 @@ This example recognizes matching braces followed by zero or more line terminator
 
 Examples of legal strings in this grammar are:
 
-`{}`, `{{{{{}}}}}` // ... etc
+`{}`, `{{{{‎{}}}}}` // ... etc
```

Comment:
There is a zero-width space between `{{{{` and `{}}}}}` to break Jekyll [Liquid](https://jekyllrb.com/docs/liquid/)'s two curly braces syntax, though not seen here